biz.aQute.bndlib.tests/src/test/VerifierPluginTest.java

```diff
@@ -0,0 +1,33 @@
+package test;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import aQute.bnd.osgi.Analyzer;
+import aQute.bnd.osgi.Builder;
+import aQute.bnd.osgi.Jar;
+import aQute.bnd.service.verifier.VerifierPlugin;
+import junit.framework.TestCase;
+
+public class VerifierPluginTest extends TestCase {
+
+	public static void testVerifierPluginExecution() throws Exception {
+		final AtomicBoolean executedCheck = new AtomicBoolean(false);
+
+		VerifierPlugin verifier = new VerifierPlugin() {
+
+			@Override
+			public void verify(Analyzer analyzer) throws Exception {
+				executedCheck.set(true);
```

Comment:
I think the test should validate that verifier is run after the analyzers, no? For example, check that manifest looks a certain way and the included resources are present in the jar.

This test just makes sure the plugin is run but not in relation to the normal analyzers.